src/Controllers/TaxonomyDirectoryController.php

```diff
@@ -20,6 +21,11 @@
  */
 class TaxonomyDirectoryController extends PageController
 {
+    /**
+     * The name of the TaxonomyTerm field to expect in the URL
+     * @config
+     */
+    private static $lookup_field = 'Name';
```

Comment:
```suggestion
    private static $lookup_field = 'Terms.Name';
```

Reply:
I changed this controller to load the `TaxonomyTerm` and get its relation to Pages instead of going from Pages that have a matching `Terms` relation. The reason was because if you're using the `cwp/cwp` module then it puts the relation on BasePage instead.

Reply:
We'd need to ensure that the changes made are backwards compatible

Reply:
So long as the backward relation `Pages` is defined then it should continue working.

Otherwise we could leave it loading from the Page class to its `Terms` relation, but anyone using `cwp` wouldn't be able to use this implementation as the `Terms` relation is defined on BasePage instead.

Alternatively there might be some way of also having the `$directory_class` configurable so that we can change it to "BasePage" in config, but is it possible to use a dynamic class's `::get()` from just the string class name? Would it need to be the full namespace `cwp\cwp\BasePage`? And then `$directory_class::get()`?

Reply:
> Is it possible to use a dynamic class's ::get() from just the string class name? Would it need to be the full namespace cwp\cwp\BasePage? And then $directory_class::get()?

My suggestion would be to require the FQN of the Page in config. Then you can use `DataObject::get($class)->filter([...])`

Reply:
Have changed as suggested so that it defaults to expecting a relation "Terms" on the Page class (as it used to), but you can override this in config e.g. for CWP to use "Terms" on the BasePage class by
```yaml
SilverStripe\Taxonomy\Controllers\TaxonomyDirectoryController:
  directory_class: CWP\CWP\PageTypes\BasePage
```